target/linux/ath79/generic/config-default

```diff
@@ -15,6 +15,8 @@ CONFIG_MARVELL_PHY=y
 CONFIG_MICREL_PHY=y
 CONFIG_MTD_REDBOOT_PARTS=y
 CONFIG_MTD_SPLIT_EVA_FW=y
+CONFIG_NVMEM_SYSFS=y
+CONFIG_NVMEM_U_BOOT_ENV=y
```

Comment:
@musashino205 I picked this nvmem commit from your original PR https://github.com/openwrt/openwrt/pull/11228 and add the `CONFIG_NVMEM_SYSFS=y`.

The entire feature increases the kernel size by about 1.5 KiB.